spec/services/session_encryptor_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe SessionEncryptor do
+  describe '#load' do
```

Comment:
that's the API for session serializer. Look at e.g. https://github.com/roidrage/redis-session-store/blob/master/lib/redis-session-store.rb#L165